modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeCatchAllCheckTest.java

```diff
@@ -70,12 +72,32 @@ public static Iterable<Object[]> data() throws Exception {
 
 			for (String fileType : fileTypes) {
 				if (_hasValidUpgradeFiles(issueKey, fileType)) {
-					objectsArray.add(new Object[] {issueKey, fileType});
+					setMap.compute(
+						issueKey,
+						(key, value) -> {
+							if (value == null) {
+								value = new HashSet<>();
+							}
+
+							value.add(fileType);
+
+							return value;
+						});
 				}
 			}
 		}
 
-		return objectsArray;
+		Set<Object[]> objectSet = new HashSet<>();
+
+		for (Map.Entry<String, Set<String>> stringSetEntry :
+				setMap.entrySet()) {
```

Comment:
Check other uses of iterating `map.getEntrySet`. Double-check the variable names for the `stringSetEntry` variable and the `setMap` variable here, whether or not they should be "generic" names (e.g. `map`, `entry`) or "purpose-built" names (e.g. `issueKeyFileTypesMap`,  `issueKeyFileTypesEntry`) or some combination of the two.

In any case, I think we should change the current variable names.